infoschema/tables.go

```diff
@@ -997,6 +997,9 @@ var tableAnalyzeStatusCols = []columnInfo{
 	{name: "FAIL_REASON", tp: mysql.TypeLongBlob, size: types.UnspecifiedLength},
 	{name: "INSTANCE", tp: mysql.TypeVarchar, size: 512},
 	{name: "PROCESS_ID", tp: mysql.TypeLonglong, size: 64, flag: mysql.UnsignedFlag},
+	{name: "Remaining_seconds", tp: mysql.TypeLonglong, size: 64, flag: mysql.UnsignedFlag},
+	{name: "Progress", tp: mysql.TypeVarchar},
```

Comment:
Don't we need a `size`?

Reply:
You are right. But it is not in the PM's design document. I think we can add this columns after communicating with PM.

Reply:
Done